recipes/deodr/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "deodr" %}
+{% set version = "0.1.14" %}
```

Comment:
It looks like `0.1.15` is up on `pypi.io`, so you might consider building from that tarball.